src/session.ts

```diff
@@ -120,6 +123,7 @@ export const patchSession = (session: WebPhoneSession): WebPhoneSession => {
     session.__reinvite = session.reinvite;
 
     session.sendRequest = sendRequest.bind(session);
+    session.sendReceive = sendReceive.bind(session);
```

Comment:
Are you sure you want to expose it?

Reply:
We handle call control features from within SDK , but this could be used to add/handle new features not built into the SDK. J team made the recommendation to expose this.

Reply:
We should only expose it if they have an immediate use case. Otherwise it could be harder to support if we keep expanding the API. Especially with low level functions.

Reply:
ok lemme make this private then for now , unless its absolutly needed. Making changes now.

Reply:
Done. Please review.